packages/brand-moment/src/BrandMoment.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode, ReactElement } from "react"
+import { Heading, Box, Paragraph } from "@kaizen/component-library"
+import { Button, ButtonProps } from "@kaizen/draft-button"
+import { SceneProps } from "@kaizen/draft-illustration"
+import { assetUrl } from "@kaizen/hosted-assets"
+import classnames from "classnames"
+import styles from "./BrandMoment.scss"
+
+type Props = {
+  backgroundColor: "blue" | "purple" | "green" | "red"
```

Comment:
Is there semantic meaning to these colours? Should `"green"` be `"success"` etc?

Reply:
From @dougmacknz: We can't do a straight `variant` prop because the colour and illustration need to be controlled independently.

We think we can trust the designer to pick the right colour and have control here.

We should cover this in documentation (which I guess we need to write)

Reply:
I've asked Marc and it does makes sense for this to be referred to by mood (positive, negative, informative) instead of colour, which also means this can be consistent with other Kaizen components like Modal, Popover, EmptyState. What's not so consistent is the name we have for this prop across other components. I've called it `variant` for now because that is the slightly more consistent one.